yarn.lock

```diff
@@ -1090,7 +1090,7 @@ block-stream@*:
   dependencies:
     inherits "~2.0.0"
 
-bluebird@^3.0.5, bluebird@^3.3.0:
+bluebird@^3.0.5, bluebird@^3.3.0, bluebird@^3.5.0:
```

Comment:
@berianjones given we shouldn't have any changes to the node dependencies then there is no reason that the lock file should be updated.